DropInDemo/Demo Base/Settings/BraintreeDemoSettings.h

```diff
@@ -1,5 +1,6 @@
 #import <Foundation/Foundation.h>
 #import <BraintreeDropIn/BTDropInRequest.h>
+#import <BraintreePaymentFlow.h>
```

Comment:
This should be either `#import <BraintreePaymentFlow/BraintreePaymentFlow.h>` or `#import "BraintreePaymentFlow.h"`.